src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1967,13 +1977,14 @@ public bool MapCurrentSrcToCompileTimeSrc(string currentSrc, out string compiler
                         char lastDirectoryChar = e.EditorPath[e.EditorPath.Length - 1];
                         if (firstFilechar == Path.DirectorySeparatorChar || firstFilechar == Path.AltDirectorySeparatorChar)
                         {
-                            file = file.Substring(1);   // Trim the directory separator
+                            file = file.Trim(new char[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar });   // Trim the directory separator(s)
                         }
                         else if (lastDirectoryChar != Path.DirectorySeparatorChar && lastDirectoryChar != Path.AltDirectorySeparatorChar)
                         {
                             continue;   // match didn't end at a directory separator, not actually a match
                         }
                         compilerSrc = Path.Combine(e.CompileTimePath, file);    // map to the compiled location
+                        compilerSrc = compilerSrc.Replace('\\', '/'); // use Unix notation for the compiled path
```

Comment:
UseUnixSymbolPaths is set to false when using local launch options on windows. When using gdbserver it will not indicate that we still need Unix style paths because we remotely compiled.